pkgs/applications/audio/virtual-ans/default.nix

```
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchurl
+, unzip
+, xorg
+, libGL
+, alsaLib
+, SDL2
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "virtual-ans-${version}";
```

Should be using `pname`
```suggestion
  pname = "virtual-ans";
```